packages/cms-editor/src/CmsEditor.tsx

```diff
@@ -51,7 +53,7 @@ function CmsEditor(props: EditorProps) {
   }
 
   if (isError) {
-    return <PanelMessage icon={IvyIcons.ErrorXMark} message={`An error has occurred: ${error.message}`} />;
+    return <PanelMessage icon={IvyIcons.ErrorXMark} message={t('error', { error: error })} />;
```

Comment:
even with so less translations, I would start with collect them together into groups. Also if the key and the variable have the same name you don't need to write error: error
```suggestion
    return <PanelMessage icon={IvyIcons.ErrorXMark} message={t('message.error', { error })} />;
```